dir.traversal.targets

```diff
@@ -33,6 +33,21 @@
     <Error Condition="'$(MSBuildLastTaskResult)'=='false'" />
   </Target>
 
+  <Target Name="GetFilesToPackage" DependsOnTargets="FilterProjects"
+          Returns="@(FilesToPackage)">
```

Comment:
Ah, never mind. I thought these targets are injected into .NET Standard projects. But these are for packaging up .NET Standard itself.